### DISCUSSION TITLE
Trying to add SSL certs to app

### DISCUSSION THREAD

**Discussion by Actticus** [fastapi/fastapi#3391] [Questions] [answered]
I get SSL certificates from certbot and trying to add them by uvicorn

`uvicorn example:app --host 0.0.0.0 --ssl-keyfile=./cert/key.pem --ssl-certfile=./cert/cert.pem`

Uvicorn working without any errors, but I can't get a page, catch ERR_EMPTY_RESPONSE. But without `--ssl-keyfile=./cert/key.pem --ssl-certfile=./cert/cert.pem` app is working. Where did i go wrong?

When I add port 443, I get ERR_CONNECTION_REFUSED

**Comment by YuriiMotov** [comment, marked as answer]
https://fastapi.tiangolo.com/deployment/concepts/#security-https
